requirements.txt

```diff
@@ -1,26 +1,26 @@
 # MISC
-PyYAML==5.4
+PyYAML==6.0
 
 # ASYNC
 uvloop==0.17.0
 
 # HTTP
-requests==2.22.0
+requests==2.28.0
 aiohttp>=3.7.4
 
 # SEARCH
-fuzzywuzzy[speedup]==0.17.0
+fuzzywuzzy[speedup]==0.18.0
 
 # API's
-discord.py[voice]==2.0.1
+discord.py[voice]==2.1.1
 
 # CHARTS
 matplotlib>=3.1.1
 pandas>=1.0.0
-emoji==0.5.4
+emoji==2.2.0
```

Comment:
That's a big update - hopefully no API changes?

Reply:
So about that...

```python
2023-02-12T00:34:48.183915+00:00 app[worker.1]: Ignoring exception in command poll-results:
2023-02-12T00:34:48.184614+00:00 app[worker.1]: Traceback (most recent call last):
2023-02-12T00:34:48.184765+00:00 app[worker.1]: File "/app/.heroku/python/lib/python3.11/site-packages/discord/ext/commands/core.py", line 229, in wrapped
2023-02-12T00:34:48.184765+00:00 app[worker.1]: ret = await coro(*args, **kwargs)
2023-02-12T00:34:48.184765+00:00 app[worker.1]: ^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-02-12T00:34:48.184767+00:00 app[worker.1]: File "/app/cogs/polls.py", line 207, in collate_poll
2023-02-12T00:34:48.184767+00:00 app[worker.1]: for _, desc, reaction in gen_poll_options(poll):
2023-02-12T00:34:48.184783+00:00 app[worker.1]: File "/app/cogs/polls.py", line 54, in gen_poll_options
2023-02-12T00:34:48.184783+00:00 app[worker.1]: key = emoji.emojize(option[:option.find(' ')], use_aliases=True)
2023-02-12T00:34:48.184783+00:00 app[worker.1]: ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-02-12T00:34:48.184802+00:00 app[worker.1]: TypeError: emojize() got an unexpected keyword argument 'use_aliases'
```

Reply:
Tight, just needed to swap `use_aliases=True` for `language='alias'`.